src/frontend/src/catalog/root_catalog.rs

```diff
@@ -373,6 +373,30 @@ impl Catalog {
             .ok_or_else(|| CatalogError::NotFound("table id", table_id.to_string()))
     }
 
+    // Used by test_utils only.
+    pub fn alter_table_name_by_id(&mut self, table_id: &TableId, table_name: &str) {
+        let (mut database_id, mut schema_id) = (0, 0);
+        let mut found = false;
+        for database in self.database_by_name.values() {
+            if !found {
+                for schema in database.iter_schemas() {
+                    if schema.iter_table().any(|t| t.id() == *table_id) {
+                        found = true;
+                        database_id = database.id();
+                        schema_id = schema.id();
+                        break;
+                    }
+                }
+            }
+        }
+
+        if found {
+            let mut table = self.get_table_by_id(table_id).unwrap();
+            table.name = table_name.to_string();
```

Comment:
A tricky problem might be also updating the SQL definition of this table and the materialized views that depend on it. 🤔 Currently adding columns to table relies on this.

Reply:
This's only used for test utils in frontend. There's no problem in meta side, columns and table name will be sequentially updated there.

Reply:
Yes, you're right. 🥵 I forgot about the definition. Will find a way to fix it tomorrow.